app/components/at_mention/at_mention.js

```diff
@@ -86,6 +87,20 @@ class AtMention extends React.PureComponent {
         };
     }
 
+    handleLongPress = () => {
+        const action = {
+            text: 'Copy Mention',
```

Comment:
If we are going to include this, then we need to add localization support for it